server/sonar-server/src/main/java/org/sonar/server/plugins/ServerExtensionInstaller.java

```diff
@@ -71,7 +72,8 @@ public void installExtensions(ComponentContainer container) {
           ExtensionProvider provider = (ExtensionProvider) container.getComponentByKey(extension);
           installProvider(container, pluginInfo, provider);
         }
-      } catch (Exception e) {
+      } catch (Throwable e) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Catch Exception instead of Throwable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1181)